### PR TITLE
[release/3.0] Add Fedora 32 and CentOS 8 runtime ids

### PR DESCRIPTION
**Pull request by omajid**
This is combined backport of:

- PR #41397: commit 6f82bd59a05ea9556397596ae9e59f39b904b222
- PR #40786: commit 86fc9e751c424278a9c4e163e0a71a9d7e30c8e7

See also: https://github.com/dotnet/corefx/pull/41549

### CODE REVIEW

**Comment by maryamariyan** [comment]
cc: @ViktorHofer

**Review by safern — APPROVED** [review]
//cc @bartonjs

**Comment by bartonjs** [comment]
### Description

New OS versions need to be added to the RID graph.

### Customer Impact:

Customers on these new OS versions may have package resolution errors due to their detected version not being parented to a known version.

### Regression? 

No, ongoing OS evolution.

### Risk

Low.  Generated code on an easy to interpret change.

### Tests run / added

Normal CI.

**Review by bartonjs — APPROVED** [review]

**Comment by omajid** [comment]
Thanks for the review and servicing proposal!

File: src/packages.builds [hunk 1]
```diff
@@ -26,6 +26,9 @@
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
     <!-- add specific builds / pkgproj's here to include in servicing builds -->
+    <Project Include="$(MSBuildThisFileDirectory)..\pkg\Microsoft.NETCore.Platforms\Microsoft.NETCore.Platforms.builds">
```
Comment (safern) [inline src/packages.builds:29]: Also, can you bump the version of the platforms package?

https://github.com/dotnet/corefx/blob/c1778515a3bee34cc09c757b5563d0af0c8b1e99/eng/Packaging.props#L29

To 3.0.1

Comment (omajid) [reply src/packages.builds:29]: Thanks. Done.

**Comment by danmosemsft** [comment]
@omajid as noted above, you do not need shiproom approval. You do however need @wtgodbe to confirm the branch is open. Is this ready to merge?

**Comment by danmosemsft** [comment]
Marking nomerge while we wait for branch

**Comment by maryamariyan** [comment]
Thank you for your contribution. As announced in dotnet/coreclr#27549 this repository will be moving to dotnet/runtime on November 13. If you would like to continue working on this PR after this date, the easiest way to move the change to dotnet/runtime is:
1.	In your corefx repository clone, create patch by running `git format-patch origin`
2.	In your runtime repository clone, apply the patch by running `git apply --directory src/corefx <path to the patch created in step 1>`

**Comment by omajid** [comment]
Hi @maryamariyan. This pull request is for the release/3.0 servicing branch. The announcement you link to says:

> The repositories will remain active for servicing changes to .NET Core 3.1 and earlier hence we will not be using the GitHub archive capability.

Do I really need to take an action here?

**Comment by maryamariyan** [comment]
@omajid no you can discard this message. thank you.

**Comment by Anipik** [comment]
@omajid can you handle the merge conflicts.

**Comment by omajid** [comment]
@Anipik I have rebased this PR to resolve conflicts. I have left `<PlatformPackageVersion>3.0.1</PlatformPackageVersion>` as it is. Please let me know if this needs to be bumped, given .NET Core 3.0.1 went out yesterday.

**Comment by safern** [comment]
3.0.1 is the right package version so yes, leave as is! Thanks, @omajid

**Review by Anipik — APPROVED** [review]

**Pull request merged**